storybook/pages/CommunitiesPortalModelEditor.qml

```diff
@@ -87,6 +56,49 @@ ListView {
             }
         }
 
+        ListView {
```

Comment:
This is a storybook page and we avoid using StatusQ components

Reply:
It has been decided to avoid using StatusQ components in storybook editors, it is well explained there: https://github.com/status-im/status-desktop/pull/7856#issue-1404470129